src/components/Dashboard/Content/index.tsx

```diff
@@ -44,7 +44,7 @@ const DashboardContent = ({
   ) : (
     <div className="flex flex-col h-full" {...attributes}>
       <div
-        style={{ backgroundImage: `url(${surveys[currentPosition].imageUrl})` }}
+        style={{ backgroundImage: `url(${surveys[currentPosition].coverImageUrl})` }}
```

Comment:
I think you can get a high-resolution image if you append the URL with `l`:

- low resolution: https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_
- high resolution: https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l

we can write a helper to do this for us - we then can use the helper here 🙌🏿

Reply:
@tyrro created in [c63ecf1](https://github.com/manh-t/react-survey/pull/55/commits/c63ecf129ba997d459fa126c73f9876f50275bb1)